internal/core/src/exec/operator/groupby/SearchGroupByOperator.h

```diff
@@ -37,44 +40,106 @@ class DataGetter {
  public:
     virtual std::optional<T>
     Get(int64_t idx) const = 0;
+
+ protected:
+    std::optional<std::string> json_path_;
 };
 
```

Comment:
As the json column may return int/string/bool for one single path, so we need to differentiate
InnerRawType and OutputType.
When group by json field:
InnerRawType=json
OutputType=inner path type specified by users